src/components/token/TokenMarkers.js

```diff
@@ -1,5 +1,5 @@
 import React, { useState, useEffect } from 'react';
-import { tokenBannedMessage } from '../../messages';
+// import { tokenBannedMessage } from '../../messages';
```

Comment:
Why did you remove this? This message is important. We won't show only the metadata reason here. This message is yet to be decided because the wording is been decided with the legal team.

Reply:
I'm removing this from token markers because this message belongs on the `TokenAlerts` component